City-of-Helsinki/helsinki-paatokset
# PP-264: Update aggregated data from Ahjo API

**To test**
- Checkout branch
- SSH into the container and run `drush ap:fs -v`
- The following migrations should have this many items available:
  - `drush migrate-status ahjo_meetings:latest`-> 114
  - `drush migrate-status ahjo_cases:all` -> 3253
  - `drush migrate-status ahjo_decisions:all`-> 4257
  - `drush migrate-status ahjo_decisionmakers:all`-> 668
  - `drush migrate-status ahjo_trustees:all` -> 391
